flake.nix

```diff
@@ -58,6 +58,7 @@
                         && app ? program
                         && builtins.isString app.program;
                       what = "app";
+                      shortDescription = app.description or "";
```

Comment:
Per https://github.com/NixOS/nix/pull/11297, this should be `app.meta.description`, right?